Python_Assignments (1).py

```diff
@@ -0,0 +1,1894 @@
+a=list('abcdabdcaadbdb')
+print(a)
+b=input("Enter any element to find out how many times the element occurred in the given list=")
+count=0
+for i in a:
+    if i==b:
+        count +=1
+print("{} occured {} times".format(b,count))
+
+
+# In[128]:
+
+
+#60. Take an element from the user and find out how many number of occurrences are there in given tuple
+a=tuple('abcdabdcaadbdb')
+b=input("Enter any element to find out how many number of occurrences are there in a given tuple=")
+count=0
+for i in a:
+    if i==b:
+        count +=1
+print("{} occured {} times".format(b,count))
+
+
+# In[155]:
+
+
+'''61. Reverse the string without effecting the special symbols. It involves three variations. Write
+code for three variations.'''
+
+s = 'ab!cd&'
+op = ''
+for i in s:
+    if i.isalnum():
+        op += i
+
+op1 = op[::-1]
+res = ''
+print(s, len(s))
+print(op1)
+j = 0
+for i in range(len(s)):
+    if not s[i].isalnum():
+        print(i, 'inside if', s[i])
+        res += s[i]
+    else:
+        if j < len(op1):
+            print(op1[j], i, j)
+            res += op1[j]
+            j += 1
+
+print(res)
+
+        
+        
+
+
+# In[140]:
+
+
+'''62.Input: abc123,#$45def6%$^789$%^, output: $%^987%$^6fed54,#$321cba'''
+
+
+# In[ ]:
+
+
+'''63.Input: abc123,#$45def6%$^789$%^, output: 9876fe,#$d54321%$^cba$%^'''
+
+
+# In[2]:
+
+
+'''64.Inout: "123,#$456%$^789$%^", Output: 321,#$654%$^987$%^
+		Only numbers has to reverse.
+ '''
+s = "123,#$456%$^789$%^"
+op = ''
+tem = ''
+for i in s:
+    if i.isdigit():
+        tem += i
+    else:
+        op += tem[::-1] + i
+        tem = ''
+print(op)
+
+
+# In[164]:
+
+
+'''65. define a function to take person details name and age are mandatory 
+parameters and height weight are optional parameters. If the user willing to 
+pass any other details(like adhar, cell, pan, passport etc..) regarding him then 
+your function should access those details.'''
+
+def func(name, age, height = None, weight = None, **kwargs):
+    print(name, age)
+    if height is not None:
+        print(height)
+    if weight is not None:
+        print(weight)
+    for i in kwargs.items():
+        print(i)
+func("John", 25, height=175, weight=70, adhar="1234567890", cell="9876543210")
+
+
+# In[4]:
+
+
+'''65a. rewrite above assignments by functions. Can use string functions to solve the string related assignments'''
+
+def even(a):
+    if a%2==0:
+        print("The given value is a even number")
+    else:
+        print("The given value is not a even number")
+n=int(input("Enter any number: "))
+even(n)
+
+
+# In[ ]:
+
+
+# 65 c. write a function to check given value is prime or not
+def prime(n):
+    if n>1:
+        for i in range(2,n//2):
+            if n%i == 0:
+                print("{} is not a prime number".format(n))
+                break
+        else:
+            print("{} is a prime number".format(n))
+    else:
+        print("{} is not a prime number".format(n))
+a=int(input("Enter any number: "))
```

Comment:
please check input is 4 it gives the output as 4 is a prime.except that input 4 only remaining inputs are working.